src/TokenForm.js

```diff
@@ -25,9 +25,10 @@ const TokenForm = () => {
         console.log('tokens: ', tokens);
         setTokens(tokens);
       });
+      return unsubscribe;
```

Comment:
I think that what Mike said is that in this code the `fetchItems` isn't called. because, when you call a function you need to call it  with `()` after the function name. And this code returns it only with the name without `()`. The right way is `fetchItems()`